src/index.js

```diff
@@ -26,6 +26,7 @@ export default function supernova(env) {
   const isUnsupportedFeature = anything.sense?.isUnsupportedFeature;
   const shouldHide = { isEnabled, isFeatureBlacklisted, isUnsupportedFeature };
   const multiUserAutomation = isEnabled?.("SENSECLIENT_IM_1855_AUTOMATIONS_MULTI_USER");
+  const isChartNavigation = isEnabled?.("ACTION_BUTTON_IM_4975_CHART_NAVIGATION");
```

Comment:
I think this could have a better name. this sounds like you can navigate inside a chart. `isGoToChartEnabled` or something similar is more descriptive IMO

Reply:
changed it to `isGoToChartEnabled`